### PR TITLE
[340] Add evaluate survey page

### PR DESCRIPTION
**Pull request by AbigailMcP**
## Changes made in this PR

- Add evaluate workload measures survey
- Links to survey from evaluate page and explore resources page

**No download file yet - changes still being made to survey**

## Guidance to review

https://trello.com/c/Gz3mzzqT/340-create-evaluate-survey-page

Screenshot 2024-02-29 at 11 26 56

### CODE REVIEW

**Comment by AbigailMcP** [comment]
[Create evaluate survey page](https://trello.com/c/Gz3mzzqT/340-create-evaluate-survey-page)

**Review by mark-pkgflow — APPROVED** [review]

File: content/workload-reduction-toolkit/evaluate-workload-measures/staff-workload-survey.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+      <p>
+        <ul>
+          <li>
+            <a
+              href="https://forms.office.com/Pages/ShareFormPage.aspx?id=yXfS-grGoU2187O4s0qC-cn26r-uTMpNqURSfi9lRcVUNEg1UTdMMllFRTM1SEVRRDJWQjE3RUU5VS4u&sharetoken=MJnNysyL44umvL8f97JA"
+              class="govuk-link">
+              Microsoft Forms - staff workload survey template
+            </a>
+          </li>
+          <li>
```
Comment (claire-hughez) [inline content/workload-reduction-toolkit/evaluate-workload-measures/staff-workload-survey.md:44]: We are no longer using Google forms, please could we delete this?

I have to delete it from the identify one in another ticket :)

File: content/workload-reduction-toolkit/evaluate-workload-measures/staff-workload-survey.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+title: Staff workload survey
+layout: /markdown_resource.html.erb
+colour: green
+---
+
+# Evaluate impact using a staff workload survey
+
+<strong class="govuk-tag">Template</strong>
+
+Repeat the [survey](/workload-reduction-toolkit/identify-workload-issues/staff-workload-survey) in the
```
Comment (claire-hughez) [inline content/workload-reduction-toolkit/evaluate-workload-measures/staff-workload-survey.md:11]: I know this is copied from the prototype, but would be intrigued to get Lauren or Ashleigh's perspective on if this bit either requires a heading or should move out the box. It looks a bit weird to me without either.

File: content/workload-reduction-toolkit/evaluate-workload-measures/staff-workload-survey.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+      <h2 class="govuk-heading-m">
+        Download the staff workload survey
+      </h2>
+      <div class="govuk-grid-row info-box__download-content">
+        <div class="govuk-grid-column-one-half">
+          <img src="/assets/images/preview-placeholder.jpg" alt="Placeholder image" class="dfe-file-preview-image">
+        </div>
+        <div class="govuk-grid-column-one-half">
+          <p class="govuk-body-s">
+            <strong>Resourced type:</strong> Template
```
Comment (claire-hughez) [inline content/workload-reduction-toolkit/evaluate-workload-measures/staff-workload-survey.md:67]:
```suggestion
            <strong>Resource type:</strong> Template
```

**Review by claire-hughez — COMMENTED** [review]
Hi Abi,

Looks great. I do have a design qu for Ashleigh/Lauren to quickly double check too which I've highlighted in the relevant bit, but if you have any thoughts, please shout.

File: content/workload-reduction-toolkit/evaluate-workload-measures/staff-workload-survey.md (hunk 2 above)
Comment (claire-hughez) [reply content/workload-reduction-toolkit/evaluate-workload-measures/staff-workload-survey.md:11]: We'll add this in another PR.

**Pull request merged**